README.md

```diff
@@ -1,3 +1,13 @@
 # verbose-squid :squid:
 # locquacious-langosta :lobster:
 ### Don't get stuck talking to either or the tide will take you away. I spoke to a cephalopod once for 16 hours and lost my shoes. I got kinda crabby :crab:. Luckily, it was during a Neap tide when the tidal range is at its smallest. 
+
+### Development Instructions
+
+This project now has a lil golang app included in it. To get it running:
+
+- Install golang (1.16 or higher is a good choice)
+
+In the root, run `go run main.go`
```

Comment:
Could you please refer to this as the project root.

Reply:
You got it!

Reply:
Change done in: 8edfee2